ssh.py

```
@@ -2,30 +2,155 @@
+                "The connection is vulnerable to man-in-the-middle attacks. "
+                "Set SSH_HOST_KEY_CHECKING=accept-new (default) or strict.",
+                self.host,
+            )
+            self.client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+            return
+
+        # Load system + managed known_hosts so previously-seen keys are trusted
+        # and any key change raises BadHostKeyException.
+        try:
+            self.client.load_system_host_keys()
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)